modules/exploits/windows/mssql/mssql_payload.rb

```diff
@@ -70,15 +70,6 @@ def initialize(info = {})
   end
 
   def check
-    if session
```

Comment:
Looks like this pattern was missed below. Added a comment to the original PR.
https://github.com/rapid7/metasploit-framework/pull/18763/files#r1515821978